csc302-spring-2021/proj-DaTeam
# feat: FormResponseValidator

## Description

- Closes #91 
- Implements FormResponseValidator validation of textfields and listfields

## Checklist

- [x] I have written unit tests for the code I added (yes, in Tests.ts)

## QA Steps



1. `cd shared && npx tsc && npm run test`
Expected outcome: No errors under the "Validator test" heading.

If you want to QA vigorously, you can create your own mock form and formresponse to validate specific errors.
